.github/workflows/release-version.yml

```diff
@@ -55,10 +55,9 @@ jobs:
           VERSION_TAG="v$(jq -r '.version' package.json)" # e.g. v1.1.0
           git add --all
           git commit -m $VERSION_TAG
-          git tag $VERSION_TAG
```

Comment:
Creating a tag will be done here:

https://github.com/DataDog/synthetics-ci-github-action/blob/ed2f8b699ee93f66b4d8d2630ba44bf0906d41d2/.github/workflows/release-version-on-merge.yml#L20-L33

Reply:
This is merged in `main` already, right?

Reply:
Yep it is!